### PR TITLE
Rename Expo read-session-user-data doc; remove unused import

### PR DESCRIPTION
**Pull request by alexisintech**
> [!WARNING]
> Need [clerk/clerk#549](https://github.com/clerk/clerk/pull/549) merged for redirects

### Explanation:

Needed to remove an unused import from an expo doc. Also found that the /expo/expo-read-session-user-data doc doesn't /expo twice.

### CODE REVIEW

**Review by jescalan — APPROVED** [review]

**Review by jescalan — CHANGES_REQUESTED** [review]
Actually hold up don't we now need to add a redirect for the old path

**Comment by alexisintech** [comment]
> Actually hold up don't we now need to add a redirect for the old path

https://github.com/clerk/clerk/pull/549

**Review by jescalan — APPROVED** [review]
Approved contingent on redirect PR merging 😁

**Pull request merged**
